modules/osdk-aws-sts-enabling.adoc

```diff
@@ -0,0 +1,323 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-token-auth.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-aws-sts-enabling_{context}"]
+= Enabling Operators to support CCO-based workflows with AWS STS
+
+As an Operator author designing your project to run on Operator Lifecycle Manager (OLM), you can enable your Operator to authenticate against AWS on STS-enabled {product-title} clusters by customizing your project to support the Cloud Credential Operator (CCO).
+
+With this method, the Operator is responsible for creating the `CredentialsRequest` object, which means the Operator requires RBAC permission to create these objects. Then, the Operator must be able to read the resulting `Secret` object.
+
+[NOTE]
+====
+Pods related to the Operator deployment mount a `serviceAccountToken` volume for the secret by default.
```

Comment:
It was in reference to the secret in the paragraph before it:
> With this method, the Operator is responsible for creating the `CredentialsRequest` object, which means the Operator requires RBAC permission to create these objects. Then, the Operator must be able to read the resulting `Secret` object.

I could make this more specific here:
```suggestion
Pods related to the Operator deployment mount a `serviceAccountToken` volume for the resulting `Secret` object by default.
```

Reply:
Okay. If this is not already done elsewhere, it would be great to mention how is the SA token used (by the resulting Secret), i.e. 

Pods related to the Operator deployment mount a ServiceAccount token as volume, so that it (the SA token) can be referenced in the resulting Secret.

Reply:
Updated to:
> By default, pods related to the Operator deployment mount a `serviceAccountToken` volume so that the service account token can be referenced in the resulting `Secret` object.